intellij/src/saros/intellij/ui/wizards/pages/referencepointselection/ReferencePointTab.java

```diff
@@ -578,23 +578,30 @@ private void updateInputValidity() {
    * TODO check for other separators? Intellij also sees '\' as a separator on unix but this is not
    *  detected by the Java Unix path implementation
    */
-  // TODO check whether resource with same name already exists in chosen base directory
   private boolean hasValidNewDirectoryName() {
-    String enteredName = newDirectoryNameTextField.getText();
-    if (enteredName.isEmpty()) {
+    String enteredName = newDirectoryNameTextField.getText().trim();
+
+    if (enteredName.isEmpty() || enteredName.contains(File.pathSeparator)) {
```

Comment:
Isn't this check also done in the following try/catch block ?

Reply:
The `isEmpty` check is technically unnecessary as it is also covered by the path check. I just included it to exit as early as possible, skipping the construction of the `Path` object.

The check whether the name contains a path separator (`;` for Windows or `:` for Unix) is not contained in the try/catch block.